tests/data/sample_specification.yaml

```diff
@@ -62,6 +60,7 @@ speech_acts:
   performative_mt:
     content_union_1: pt:union[ct:DataModel, pt:bytes, pt:int, pt:float, pt:bool, pt:str, pt:set[pt:int], pt:list[pt:bool], pt:dict[pt:str, pt:int]]
```

Comment:
Use a different name to DataModel.

Reply:
Also add `pt:list[ct:DataModel]` and `pt:set[ct:DataModel]` (with different names for DataModel) inside of the union.

Reply:
done